.github/CODEOWNERS

```
@@ -1499,6 +1499,7 @@ x-pack/test/security_solution_api_integration/test_suites/entity_analytics @elas
 x-pack/test/security_solution_cypress/cypress/e2e/entity_analytics @elastic/security-entity-analytics
 x-pack/plugins/security_solution/public/flyout/entity_details @elastic/security-entity-analytics
 x-pack/plugins/security_solution/common/api/entity_analytics @elastic/security-entity-analytics
+x-pack/plugins/security_solution/server/search_strategy/security_solution/factory/risk_score
```

Did you intend for this code ownership to be assigned to the security-entity-analytics team?
```suggestion
x-pack/plugins/security_solution/server/search_strategy/security_solution/factory/risk_score @elastic/security-entity-analytics
```

---

oops. Thank you!